docs/adr/2022-12-05_005-remove-ptr-addresses.md

```diff
@@ -0,0 +1,37 @@
+---
+slug: 5
+title: |
+  5. Remove support for pointer addresses
+authors: [Jared Corduan]
+tags: [Accepted]
+---
+
+## Status
+
+Accepted
+
+## Context
+
+[CRC-2](https://github.com/cardano-foundation/CIPs/pull/374) provides context.
+
+The removal of pointer addresses is needed for
+[Ouroboros Leios](https://iohk.io/en/research/library/papers/ouroboros-leios-design-goals-and-concepts/).
+
+## Decision
+
+We will remove support for pointer addresses.
+
+## Consequences
+
+Pointer addresses will no longer be available.
+The existing ~11 pointer addresses will be converted to enterprise addresses.
```

Comment:
I forgot if this document is  meant to be a definitive decision?  And also I forgot if we were totally decided on this part, I thought maybe there were some problems with it? 
But anyway, i seem to don't remember well, so please ignore me if that's the case!

Reply:
yes, this is suppose to reflect a decision that we all agreed to.

I think we are all in agreement that this is the plan, but I should make a note of the fact that we are doing it in two stages!
1. prevent the creation of pointer addresses
2. later (after a hard fork) remove pointer addresses everywhere